app/models/binary/Publication.scala

```diff
@@ -87,8 +89,6 @@ class PublicationDAO @Inject()(sqlClient: SQLClient)(implicit ec: ExecutionConte
     for {
       _ <- run(
         sqlu"""insert into webknossos.publications(_id, publicationDate, imageUrl, title, description, created, isDeleted)
-                         values(${p._id.id}, ${p.publicationDate
-          .map(new java.sql.Timestamp(_))}, ${p.imageUrl}, ${p.title}, ${p.description}, ${new java.sql.Timestamp(
-          p.created)}, ${p.isDeleted})""")
+                         values(${p._id.id}, ${p.publicationDate}, ${p.imageUrl}, ${p.title}, ${p.description}, ${p.created}, ${p.isDeleted})""")
```

Comment:
I assume the previous `map` is not needed anymore since publicationDate is an Option and can be deserialized recursively (e.g. first the Option deserialization, if there's a value the Instant is deserialized), is this correct?

Reply:
Almost correct! It’s not exactly recursively, but SQLTypeImplicits now has `SetInstantOpt` as well as `SetInstant`